pyproject.toml

```diff
@@ -178,12 +175,12 @@ check = [
 
 [tool.hatch.envs.style.scripts]
 check = [
-  "black --check --diff --color .",
   "ruff .",
+  "ruff format --check --diff .",
 ]
 fix = [
-  "black .",
   "ruff --fix .",
+  "ruff format .",
```

Comment:
I think in general format should go first before `fix`. For example, in the future, format may be able to re-format lines that are too long but `fix` may not be able to.